jwst/photom/photom.py

```diff
@@ -796,6 +803,12 @@ def photom_io(self, tabdata, order=None):
                     conversion, no_cal = self.create_2d_conversion(self.input.slits[self.slitnum],
                                                                    self.exptype, conversion,
                                                                    waves, relresps, order)
+            elif isinstance(self.input, datamodels.MultiSpecModel):
+
+                # This input does not require a 2d conversion, just an application of the
+                # relresponse to the DN/s input.
```

Comment:
Not a big deal, but the wording of this comment makes me think - at first pass - that these data don't need converting/calibration (at all). But it's not that they don't need conversion, it's that they don't need the conversion applied to *2D* data that's the important part. So maybe something more along the lines of "This input is already 1d and hence only needs to have the relresponse applied directly to the input without interpolation to 2d space".